stdlib/REPL/src/REPL.jl

```diff
@@ -432,9 +448,10 @@ mutable struct LineEditREPL <: AbstractREPL
     mistate::Union{MIState,Nothing}
     interface::ModalInterface
     backendref::REPLBackendRef
+    last_shown_line_infos::Vector{Tuple{String,Int}}
     LineEditREPL(t,hascolor,prompt_color,input_color,answer_color,shell_color,help_color,history_file,in_shell,in_help,envcolors) =
         new(t,hascolor,prompt_color,input_color,answer_color,shell_color,help_color,history_file,in_shell,
-            in_help,envcolors,false,nothing, Options(), nothing)
+            in_help,envcolors,false,nothing, Options(), nothing, Tuple{String,Int}[])
```

Comment:
It seems that this object is partially initialized by default. The number of arguments to `new` is less than the number of fields. So, it might be adding the default argument of an incompatible type. Not sure what's the best way to fix it, though. The simplest solution seems to put `last_shown_line_infos` somewhere above `interface`.

Reply:
Thanks so much, that seems to have fixed it. I really did not think of that.